smartobjects-net-client/Impl/CredentialHandler.cs

```
@@ -109,7 +109,7 @@ private void RequestToken()
             }
             catch (Exception)
             {
-                throw new InvalidOperationException("Error fetching token...");

```

Isn't this change potentially a breaking one?

---

It could be if the user is matching on this very specific Exception class.

I'll remove the change, but we should avoid swallowing the source exception like that.

---

Agreed.